src/drivers/mct/main/seq_hist_mod.F90

```diff
@@ -862,16 +854,18 @@ subroutine seq_hist_writeavg(infodata, EClock_d, &
              end if
 
              avg_time = 0.5_r8 * (tbnds(1) + tbnds(2))
+
              !---------- tcx nov 2011 tbnds of same values causes problems in ferret
-             if (tbnds(1) >= tbnds(2)) then
+             if (tbnds(1) == tbnds(2)) then
```

Comment:
This if block was backward.

Reply:
I'm not sure I'd say it was backward, but your change seems okay. My understanding is that, in general, we have either:
- tbnds(1) < tbnds(2) – the normal case
- tbnds(1) == tbnds(2) – the case this conditional is trying to avoid

I can see arguments either way for how to handle the case where tbnds(1) > tbnds(2), but I doubt it ever happens in practice.